scipy/signal/_peak_finding.py

```diff
@@ -313,6 +418,10 @@ def peak_prominences(x, peaks, wlen=None):
     of peaks in `x`. This behavior may even be used intentionally to calculate
     "local" prominences.
 
+    For indices in `peaks` that don't point to valid local maxima in `x` the
```

Comment:
I'll change it. Is this documented somewhere? The dev-guide points to [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#sections) which supports the `Warnings` section. Because I didn't see it anywhere else in SciPy I did it his way.

Reply:
If it's in numpydoc docs, it's fine I think.

Reply:
+1 for `Warnings` section